chemclipse/tests/org.eclipse.chemclipse.numeric.fragment.test/src/org/eclipse/chemclipse/numeric/statistics/Calculations_17_Test.java

```diff
@@ -71,7 +65,7 @@ public void testScaleToStandardizedLength_1() {
 		standardizedLength[11] = -1.291943343966863d;
 		standardizedLength[12] = -0.6159186250380443d;
 		for(int i = 0; i < values.length; i++) {
-			assertEquals("StandardizedLengthValues", standardizedLength[i], values[i]);
+			assertEquals("StandardizedLengthValues", standardizedLength[i], values[i], 0);
```

Comment:
Consider replacing the for cycle with assertArrayEquals(standardizedLength, values).